docs/models/regressor.md

```diff
@@ -0,0 +1,21 @@
+Supported languages
+------------------
+
+The regressor model supports other languages too (all languages supported by rust-code-analysis: https://github.com/mozilla/rust-code-analysis#supported-languages).
+
+Training the model for another project
+--------------------------------------
+
+There are quite a few steps to reproduce the results on another project, and they kind of depend on the processes followed by the specific project. Here is the current pipeline, which depends on Mozilla's processes. Some steps might me not necessary for other projects (and some projects might require additional steps).
+
+1. Gather bugs from the project's Bugzilla; 
```

Comment:
There's a trailing whitespace here that is making linting fail

Reply:
done